iothub/service/src/Registry/Models/StorageAuthenticationType.cs

```diff
@@ -16,13 +15,11 @@ public enum StorageAuthenticationType
         /// Use a shared access key for authentication.
         /// </summary>
         /// <remarks>This means authentication must be supplied in the storage URI(s).</remarks>
-        [EnumMember(Value = "keyBased")]
-        KeyBased = 0,
+        KeyBased,
```

Comment:
Q - should all enums require an "unknown/none" option that they should default to?

Reply:
I like that idea, but I don't feel compelled to do that in this PR.